modeshape-jcr/src/main/java/org/modeshape/jcr/spi/index/provider/IndexChangeAdapters.java

```diff
@@ -803,6 +803,11 @@ public NodeTypesChangeAdapter( String property,
             assert this.property != null;
         }
         
+        private static Object[] valuesOf(Property property) {
+            return property.getValuesAsArray() != null ? 
+                    property.getValuesAsArray() : new Object[]{};
```

Comment:
I think `new Object[0]|` is more concise, since an empty array initializer doesn't make sense